python/tvm/relay/op/nn/nn.py

```diff
@@ -698,6 +698,29 @@ def softmax(data, axis=-1):
     return _make.softmax(data, axis)
 
 
+def fast_softmax(data, axis=-1):
+    r"""Computes softmax.
+    Use approximation to compute exponent for faster speed.
+
+    .. math:: \text{softmax}(x)_i = \frac{exp(x_i)}{\sum_j exp(x_j)}
```

Comment:
would you like to update the formula to reflect we are using alternative ways to compute the softmax?

Reply:
The math formula is still the same, but we use some approximation to compute exp.
The approximation cannot be described as short formulas, so we only mentioned this by text in the doc string at L703.

Reply:
Okay I am not going to block this.